src/test/java/org/elasticsearch/index/mapper/geo/GeoPointFieldMapperTests.java

```diff
@@ -413,4 +468,31 @@ public void testLonLatArrayArrayStored() throws Exception {
         assertThat(doc.rootDoc().getFields("point.lon")[1].numericValue().doubleValue(), equalTo(1.5));
         assertThat(doc.rootDoc().getFields("point")[1].stringValue(), equalTo("1.4,1.5"));
     }
+
+    @Test
+    public void testGeoPointMapperMerge() throws Exception {
+        String stage1Mapping = XContentFactory.jsonBuilder().startObject().startObject("type")
+                .startObject("properties").startObject("point").field("type", "geo_point").field("lat_lon", true).field("geohash", true)
+                .field("validate", true).field("normalize", true).endObject().endObject()
+                .endObject().endObject().string();
+        DocumentMapperParser parser = createIndex("test").mapperService().documentMapperParser();
+        DocumentMapper stage1 = parser.parse(stage1Mapping);
+        String stage2Mapping = XContentFactory.jsonBuilder().startObject().startObject("type")
+                .startObject("properties").startObject("point").field("type", "geo_point").field("lat_lon", true).field("geohash", true)
+                .field("validate", false).field("normalize", false).endObject().endObject()
```

Comment:
correct. normalize will be changing in a separate PR. I must have accidentally included it in this one. :)  Made changes.